include/modem/modem_info.h

```diff
@@ -57,14 +48,70 @@ extern "C" {
 /** SNR offset value. */
 #define SNR_OFFSET_VAL 24
 
-/** Modem returns RSRP and RSRQ as index values which require
- * a conversion to dBm and dB respectively. See modem AT
- * command reference guide for more information.
+/** @brief Converts RSRP index value returned by the modem to dBm.
+ *
+ * The index value of RSRP can be converted to decibel with the following formula:
```

Comment:
This should be dBm instead, but should it be plural, @peknis?
```suggestion
 * The index value of RSRP can be converted to dBm with the following formula:
```

Reply:
This is copy paste from AT command spec and I did send the question to Jaakko earlier

Reply:
Can be either singular or plural. Not a big deal.

Reply:
I'll use dBm in the text because decibel from AT spec is just wrong.